### PR TITLE
Add NHC products (afos,grib_stats.short,stats.tpc)

### LINKED ISSUE #140: Add NHC ASCII products
## Description
Provide a clear and concise description of the requested feature/capability.
Add HAFS NHC ASCII products like:
- storm_id.yyyymmddhh.hafs.afos
- storm_id.yyyymmddhh.hafs.stats.short
- storm_id.yyyymmddhh}.hafs.stats.tpc

## Proposed solution
How should the new feature/capability be added? If you have thoughts on the implementation strategy, please share them here.
Follow HMON's method to generate these files based on the atcfunix track file.

## Status (optional)
Do you (or a colleague) plan to work on adding this feature?
@LinZhu-NOAA will explore to add this capability in HAFS application.

## Related to (optional)
Directly reference any related issues or PRs in this or other repositories, and describe how they are related. Examples:
- fixed by hafs-community/hafs/pull/156

### PR DESCRIPTION
**Pull request by LinZhu-NOAA**
## Description of changes
Add NHC products (afos,grib_stats.short,stats.tpc) for HAFS.

## Issues addressed
- fixes hafs-community/HAFS/issues/140

## Tests conducted
This change will not affect the results or other products.

### CODE REVIEW

File: scripts/exhafs_product.sh [hunk 1]
```diff
@@ -271,6 +272,36 @@ fi
...
+
+set +e
+set -o pipefail
+time ./hafs_nhc_products.x  2>&1 | tee ./hafs_nhc_products.out
+set +o pipefail
+set -e
+
+short=${out_prefix}.grib.stats.short
+afos=${out_prefix}.afos
+tpc=${out_prefix}.stats.tpc
```
Comment (BinLiu-NOAA) [inline scripts/exhafs_product.sh]: @LinZhu-NOAA, could you update these three lines into:
short=${out_prefix}.hafs.grib.stats.short
afos=${out_prefix}.hafs.afos
tpc=${out_prefix}.hafs.stats.tpc

Currently, all HAFS products in com dir have the naming convention of "${out_prefix}.hafs.*". In the future, if we come up new updated convention, we can further update this. Thanks!

**Comment by LinZhu-NOAA** [comment]
@BinLiu-NOAA The file names have been updated.

**Review by BijuThomas-NOAA — APPROVED** [review]

**Review by panll — APPROVED** [review]

**Comment by LinZhu-NOAA** [comment]
@BinLiu-NOAA The source codes re-organization is done.

**Review by BinLiu-NOAA — APPROVED** [review]
Looks good to me. Thanks for adding these nhc products in HAFS application/workflow!

**Pull request merged**